pkg/setup/network.go

```diff
@@ -274,11 +258,16 @@ func (e *Environment) setupNetwork() error {
 		return err
 	}
 
-	if isDockerBridge() {
-		e.isDockerBridge = true
-		return nil
+	err = e.generateCNIConf()
+	if err != nil {
+		return err
 	}
-	e.isDockerBridge = false
 
+	// docker set a drop by default
+	b, err := exec.Command("iptables", "-P", "FORWARD", "ACCEPT").CombinedOutput()
```

Comment:
If you know the name of the CNI bridge you can add explicitly add it to the FORWARD chain with ACCEPT (look at the docker0 config)